src/arrayMethodSort.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `__proto__` to extend the prototype is not recommended. Consider using `Array.prototype.sort2` instead for better performance and security.